src-clean/server.js

```diff
@@ -1,27 +1,39 @@
 /* eslint-disable no-console */
+
+// Necessary for SSR
+import 'isomorphic-fetch'
```

Comment:
I'm sure you've [looked it up](https://github.com/matthew-andrews/isomorphic-fetch) by now, but: isometric fetch adds support for fetch both on the client and the server, in the most platform-appropriate way. Many React/Redux libraries require a working fetch(), and doing SSR means you need to use node-fetch - or isometric-fetch which solves the issue for you.